src/types.c

```diff
@@ -85,6 +86,7 @@ void init_types(void)
     }
 
     strcpy(global_state.integers[8][false].type.name, "byte");
+    strcpy(global_state.integers[16][false].type.name, "short");
```

Comment:
The signed type should be named `short`, not the unsigned type:

```suggestion
    strcpy(global_state.integers[16][true].type.name, "short");
```